IMAGE_SIZE_ANALYSIS.md

```
@@ -0,0 +1,245 @@
+RUN yarn install --production --ignore-scripts --prefer-offline
+```
+
+### 2. **Remove Documentation & Metadata Files** (Est. savings: 20-30 MB)
+
+**Problem:** 11,584+ unnecessary files present:
+- Source maps (`.map` files)
+- README/CHANGELOG/LICENSE files in node_modules
+- Markdown documentation (55 files in root alone)
+
+**Evidence:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)